rebasehelper/tests/test_output_tool.py

```diff
@@ -131,21 +137,24 @@ def get_expected_json_output():
             },
             ResultsStore.RESULTS_PATCHES: {
                 "deleted": ["mytest2.patch"]
+            },
+            ResultsStore.RESULTS_SUCCESS: {
+                "success": "Success"
             }
         }
         return expected_output
 
     def test_text_output_tool(self, results_file_path, results_store):
-        output = OutputTool('text')
-        output.print_information(results_file_path, results_store)
+        assert TextOutputTool.get_name() in output_tools_runner.output_tools
+        TextOutputTool.print_summary(results_file_path, results_store)
```

Comment:
OK, the test will fail if there is an exception, but that's not the point of unit tests, this kind of thing should go to functional tests.